baryshev_code_HW7.py

```diff
@@ -0,0 +1,80 @@
+from pydrive.auth import GoogleAuth
```

Comment:
You might consider adding some documentation detailing what this code aims to accomplish and any requirements it might have.